external/kafka-0-10-sql/src/main/scala/org/apache/spark/sql/kafka010/KafkaOffsetReader.scala

```diff
@@ -26,7 +26,7 @@ import scala.concurrent.{ExecutionContext, Future}
 import scala.concurrent.duration.Duration
```

Comment:
There are several comments in this file which states similar things:
`This only queries the offsets and never commits them.`

Reply:
I guess I should remove them